packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/e2e/fixture.ts

```diff
@@ -9,8 +9,11 @@ import {useCaseEnum} from '../../../../../../playwright/utils/useCase';
 import {GeneratedAnswerObject} from './pageObject';
 import genQaData from './data';
 import type {GenQaData} from './data';
+import {AnalyticsMode} from '../../../../../../playwright/utils/analyticsMode';
+import {AnalyticsHelper} from '../../../../../../playwright/page-object/analytics';
 
 const pageUrl = 's/quantic-generated-answer';
+const exampleTrackingId = '1234';
```

Comment:
pourrais etre une constante dans le analytics helper?

Reply:
Yea good idea, also as this is events that will actually reach an org right?

It would be easier to use a value to easily identify the source of the events.